oak_abi/src/label/tests.rs

```diff
@@ -57,8 +57,8 @@ fn serialize_deserialize() {
 
 #[test]
 fn label_flow() {
-    let tag_0 = authorization_bearer_token_hmac_tag(&[0, 0, 0]);
-    let tag_1 = authorization_bearer_token_hmac_tag(&[1, 1, 1]);
+    let tag_0 = public_key_identity_tag(vec![0, 0, 0]);
```

Comment:
Is there a particular reason why the other fn takes a slice ref, and the new one an actual vec? May be worth changing it to take a ref as well, perhaps in a separate PR, but up to you anyways.